gaphor/ui/filemanager.py

```diff
@@ -326,6 +359,31 @@ def response(answer):
             confirm_shutdown()
 
 
+def merge_conflict_dialog(window: Gtk.Window, filename, handler) -> None:
+    dialog = Adw.MessageDialog.new(
+        window,
+        gettext("Resolve merge conflict in model “{filename}”.").format(
```

Comment:
Maybe it should say Resolve Merge Conflict?

Reply:
I changed the name to `resolve_merge_conflict_dialog`.

Reply:
This is what I meant, the actual heading of the message dialog. 
![image](https://user-images.githubusercontent.com/10014976/209970705-d5103aa6-a027-426c-a5cc-24c340a3cb2f.png)

Reply:
Ah. That's a lot nicer!

Reply:
fixed